Automattic/jetpack
# VideoPress: Add ThemeProvider to Dropdown and Popover components

Fixes #33808

## Proposed changes:

* Adds `ThemeProvider` wrapper on `Dropdown` and `Popover` child content
* Fix nested styles to work with the portal

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On `trunk` check that the style issue happens only when Gutenberg >= 16.6 is installed and activated
* On this branch, check that popovers and dropdowns work on the VideoPress dashboard with Gutenberg >= 16.6
* Check that there is no change when Gutenberg is deactivated

Specific versions of Gutenberg can be downloaded from https://github.com/WordPress/gutenberg/releases

| Before | After
| - | -
| ![2023-10-26_13-44-30](https://github.com/Automattic/jetpack/assets/8486249/a9704736-9dc5-45df-a0a7-5a9c5e57bc94) ![2023-10-26_13-42-08](https://github.com/Automattic/jetpack/assets/8486249/b8e8ff43-61f0-4486-ab3d-48a7a85172ad) |  ![2023-10-26_17-11-33](https://github.com/Automattic/jetpack/assets/8486249/56e0f854-cc4a-4bf8-8291-ae924f189c3d) ![2023-10-26_17-11-51](https://github.com/Automattic/jetpack/assets/8486249/ee582af7-123b-417e-9289-817ddd2bbafd)
